README.md

```diff
@@ -287,6 +289,7 @@ terraform destroy
 
 | Name | Description |
 |------|-------------|
+| bla |  |
```

Comment:
Could me removed I guess :)

Reply:
@npalm  Yeah, right, during debugging I added the bla output, will remove it!